io_unix.go

```diff
@@ -26,8 +26,8 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-// NewPipeIO creates pipe pairs to be used with runc
-func NewPipeIO(uid, gid int, opts ...IOOpt) (i IO, err error) {
+// newPipeIO creates pipe pairs to be used with runc
+func newPipeIO(uid, gid int, opts ...IOOpt) (i IO, err error) {
```

Comment:
Let me know if we should instead make whole of `io**.go` excluded on Windows (mostly concerned if there's code that might refer the types somewhere)